.github/workflows/tests.yml

```diff
@@ -28,8 +28,14 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - run: pip install tox
 
-    - name: run Python tests
-      run: tox -e tests
+    - name: run Python metatensor-core tests
+      run: tox -e metatensor-core-numpy-tests
+
+    - name: run Python metatensor-core tests
+      run: tox -e metatensor-core-torch-tests
+
+    - name: run Python metatensor-torch tests
+      run: tox -e metatensor-torch-tests
```

Comment:
I think you can remove the `metatensor` from the name. We don't have tests without metatensor I assume.